packages/edit-site/src/components/header-edit-mode/style.scss

```diff
@@ -29,6 +29,11 @@
 			// the toolbar to grow outside of the allowed container space.
 			overflow: hidden;
 		}
+
+		&.is-opened {
+			// When top toolbar is engaged and should expand fully.
+			flex-basis: 100%;
+		}
```

Comment:
Instead of overriding the flex-basis, can we apply styles to the central area to center itself? That might make the code simpler to understand.

Reply:
It's the start area that needs to omit its flex-basis. The center area does not render when the toolbar is opened.

Reply:
And the end as well.

Reply:
Tried it here; may be better terminology? https://github.com/WordPress/gutenberg/pull/61118/commits/dc979eabfb34e77f885bf7e5cefeec52b98cdd0b

Reply:
`show-block-toolbar` is what I have currently.